test/ttmlir/Conversion/StableHLOToTTIR/dot_general/dot_general_lregression.mlir

```diff
@@ -0,0 +1,32 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
+module @jit_loss attributes {mhlo.num_partitions = 1 : i32, mhlo.num_replicas = 1 : i32} {
+  func.func public @main(%arg0: tensor<2x1xf32> {mhlo.layout_mode = "default", mhlo.sharding = "{replicated}"}, %arg1: tensor<f32> {mhlo.layout_mode = "default", mhlo.sharding = "{replicated}"}, %arg2: tensor<127x2xf32> {mhlo.layout_mode = "default", mhlo.sharding = "{replicated}"}, %arg3: tensor<127x1xf32> {mhlo.layout_mode = "default", mhlo.sharding = "{replicated}"}) -> (tensor<2x1xf32> {jax.result_info = "[0]", mhlo.layout_mode = "default", mhlo.sharding = "{replicated}"}, tensor<f32> {jax.result_info = "[1]", mhlo.layout_mode = "default", mhlo.sharding = "{replicated}"}) {
```

Comment:
Do we need all of these attributes? I believe we don't use any of these informations during lowering.